tests/end-to-end.py

```diff
@@ -1159,13 +1150,61 @@ def __init__(
     self._image_export_path = None
     self._InitializeImageExportPath()
 
+  def _CompareHashesFile(self, test_definition, temp_directory):
```

Comment:
This was already added with #3102

Reply:
For some reason github didn't show the branch needed to be synced until after my comment

Reply:
Noworries, thx for flagging